OpenLIFUSonicationPlanner/OpenLIFUSonicationPlanner.py

```diff
@@ -152,6 +171,94 @@ def setParameterNode(self, inputParameterNode: Optional[OpenLIFUSonicationPlanne
             # Note: in the .ui file, a Qt dynamic property called "SlicerParameterName" is set on each
             # ui element that needs connection.
             self._parameterNodeGuiTag = self._parameterNode.connectGui(self.ui)
+            self.addObserver(self._parameterNode, vtk.vtkCommand.ModifiedEvent, self.checkCanPlan)
+        self.checkCanPlan()
```

Comment:
does this belong in `setParameterNode` or up in `setup`?

Reply:
i moved it to `setpu` and also call it after `updatecombobox`